spec/e2e/encrypt_e2e_spec.rb

```diff
@@ -13,6 +13,19 @@
       decryptResult = Evervault.decrypt(encryptResult)
```

Comment:
we probably should test both curves here since we're supporting both!

Reply:
I've been speaking to @thomascullen about this! It's not immediately possible with the way our tests are configured but I think he's working on it

Reply:
Mainly focused on the unit tests at the moment but will be taking a look at the e2e tests after. In the meantime you might be able to just test the Evervault::Client class directly instead of the Evervault module classes.

```rb
client = Evervault::Client.new(app_uuid: app_id, api_key: api_key, curve: "koblitz")
encryptResult = client.encrypt(payload)
decryptResult = client.decrypt(encryptResult)
expect(decryptResult).to eq(payload)
```

As far as I can see there isn't actually any way to configure the curve on the Evervault module is there?

Reply:
There's not, no, that's what I got caught by. Will try the above 👍